File: fizzbuzz.py
def fizz(n,str):
	if(n%3 == 0):
		str += "Fizz"
	return n,str

def buzz(n,str):
	if(n%5 == 0):
		str += "Buzz"
	return n,str

def puts(n,str):
	if(str == ""):
		print(n)
	else:
		print(str)

if __name__ == '__main__':
	for n in range(31):
		puts(*buzz(*fizz(n,"")))
        

File: fizzbuzz.rb
class FizzBuzz
  module IntegerExtension
    refine Integer do
      def fizzBuzz
        return 'FizzBuzz' if self % 3 == 0 && self % 5 == 0
        return 'Fizz' if self % 3 == 0
        return 'Buzz' if self % 5 == 0
        self
      end
    end
  end
  using IntegerExtension

  def Main
    (1..30).each do |i|
      puts i.fizzBuzz
    end
  end
end

FizzBuzz.new.Main
